web/gds-admin-ui/src/helpers/api/apiCore.js

```diff
@@ -37,17 +40,24 @@ axios.interceptors.response.use(
     }
 );
 
-const AUTH_SESSION_KEY = 'hyper_user';
-
 /**
  * Sets the default authorization
  * @param {*} token
  */
 const setAuthorization = (token) => {
-    if (token) axios.defaults.headers.common['Authorization'] = 'JWT ' + token;
+    if (token) axios.defaults.headers.common['Authorization'] = 'Bearer ' + token;
     else delete axios.defaults.headers.common['Authorization'];
 };
 
+const setCookie = (cookie) => {
+    if (cookie) {
+        axios.defaults.headers['X-CSRF-Token'] = cookie
+    }
+}
```

Comment:
Using the pattern from `setAuthorization` may I ask why this isn't:

```javascript
const setCookie = (cookie) => {
    if (cookie) axios.defaults.headers.common['X-CSRF-Token'] = cookie;
    else delete axios.defaults.headers.common['X-CSRF-Token'];
}
```

e.g. what is `headers.common`? and do we need the functionality to delete the cookie?

Reply:
`headers.common` is to set `axios` default instance
I can do `setCookie('')` when I want to make a request without cookie but it will remove the CSRF protection

Reply:
Ok, let's keep it for now, but I think we may need the functionality in the future. Adding the csrf_token to all requests shouldn't hurt anything.